.jshintrc

```diff
@@ -0,0 +1,91 @@
+{
```

Comment:
why are you integrating jshintrc? no please
https://github.com/practo/javascript/blob/master/guides/coding-standards/tutorial.md